src/networking.c

```diff
@@ -209,6 +210,29 @@ cc_oci_tap_create(const gchar *const tap) {
 	return ret;
 }
 
+gboolean
+is_interface_ovs(struct cc_oci_net_if_cfg* if_cfg) {
```

Comment:
It should be possible to add a test for this function. To make that easier, you could define a global variable rather than using `OVS_PORT_PATH` in the function. A nicer approach would be to pass in that value though (since the tests can pass in a temporary file (`g_mkstemp()` for example) to represent the path.